### ISSUE TITLE
How do you handle twill blocks in vue.js?

### ISSUE THREAD

**Issue by andevit** [Codivores/tutorial-laravel-twill-inertia-vue3-vite-tailwind#2]
Hi there! Thanks for the tutorial!

How do you render blocks in vue.js? Do you have any examples?

**Comment by tech-codivores** [comment]
Hey @andevit,
I just updated the repository with the adaptations to make blocks working "smoothly" on both Twill and Vue sides.

I also published two articles explaining how it's done. The first one covers the concept with an example block (https://tech.codivores.com/ltivt-10-laravel-twill-inertia-vue3-bridging-twill-vue-blocks), and the second one delves into a list of blocks, which I'm continuously expanding (https://tech.codivores.com/ltivt-11-laravel-twill-vue3-block-party).

Feel free to give me feedback, as there's plenty of room for improvement, and it might be a bit unclear at times.
